### PR TITLE
kv: add unit test for BufferBatchGetter

### PR DESCRIPTION
**Pull request by coocood**
### What problem does this PR solve?

Problem Summary: improve test coverage

### What is changed and how it works?

- Add unit test
- Remove duplicate TestingT

### Check List 

Tests 

- Unit test

### CODE REVIEW

**Comment by coocood** [comment]
@cfzjywxk @lysu PTAL

File: kv/mem_buffer_test.go [hunk 1]
```diff
@@ -231,6 +231,37 @@ func (s *testKVSuite) TestBufferLimit(c *C) {
...
+	// middle value is the same as snap
+	middle := NewMemDbBuffer(4096)
+	middle.Set(ka, []byte("a1"))
+	middle.Set(kc, []byte("c1"))
+
+	buffer := NewMemDbBuffer(4096)
+	buffer.Set(ka, []byte("a2"))
+	buffer.Delete(kb)
+	r, err := buffer.Get(context.Background(), kb)
+	fmt.Println(r, err)
```
Comment (cfzjywxk) [inline kv/mem_buffer_test.go]: The print should be removed?

**Comment by cfzjywxk** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**